src/linkml/include_participant.yaml

```diff
@@ -226,13 +235,31 @@ slots:
     title: Race
     range: enum_race
     required: true
+  related_participant:
+    definition_uri: include:related_participant
+    description: Participant related to Subject Participant
```

Comment:
I might add a bit more detail here, e.g. "the family member to which the Participant is related".  (The phrase 'subject participant might be confusing to those not thinking in terms of a relationship having a subject and object).